introduction/L_community.md

```diff
@@ -2,11 +2,11 @@ The Elixir and Phoenix communities are friendly and welcoming. All questions and
 
 There are a number of places to connect with community members at all experience levels.
 
-* We're on freenode IRC in [\#elixir-lang](http://webchat.freenode.net/?channels=elixir-lang) channel.
-* We have a [Slack channel](https://elixir-slackin.herokuapp.com/).
-* The Phoenix repo has an [issue tracker](https://github.com/phoenixframework/phoenix/issues).
+* We're on Freenode IRC in the [\#elixir-lang](http://webchat.freenode.net/?channels=elixir-lang) channel.
+* [Request an invitation](https://elixir-slackin.herokuapp.com/) and join the #phoenix channel on [Slack](https://elixir-lang.slack.com).
 * For general Phoenix questions, email the [phoenix-talk mailing list](http://groups.google.com/group/phoenix-talk).
 * To discuss new features in the framework, email the [phoenix-core mailing list](http://groups.google.com/group/phoenix-core).
-* Ask or answer questions about Phoenix on [stackoverflow](http://stackoverflow.com/questions/tagged/phoenix-framework).
+* Read about [bug reports](https://github.com/phoenixframework/phoenix/blob/master/CONTRIBUTING.md#bug-reports) and  open an issue in the Phoenix [issue tracker](https://github.com/phoenixframework/phoenix/issues).
```

Comment:
What would you think about changing the "and" here to an "or"? Also, I think there's an extra space after the "and".

Read about bug reports or open an issue . . .